### PR TITLE
UG update

### PR DESCRIPTION
**Pull request by yunpeng1234**
Based on @jackgugz comment on #242.
Changes most of the thing requested.

Suggestion change to both APPLICATION_STATUS changed to both application status instead.

UI picture updated to have no academic information shown

### CODE REVIEW

**Review by jackgugz — COMMENTED** [review]
I changed my comments into todo form please tick accordingly to ensure all are done.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -370,12 +371,12 @@ If your changes to the data file makes its format invalid, Intern Watcher will d
...
 **List** | `list`
 **View** | `view INDEX [T]` <br> e.g. `view 2 T`
-**Filter** | `filter [g/GRADE] [i/INSTITUTION]…+ [c/COURSE]…+ [y/GRADUATION_YEAR_MONTH] [j/JOB] [a/STATUS] [s/SKILL]…+` <br> e.g. `filter s/JAVA y/06/2022 i/NUS a/REJECTED`
-**Edit** | `edit ALL a/APPLICATION_STATUS`/`edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [g/GRADE] [i/INSTITUTION] [c/COURSE] [y/GRADUATION_YEAR_MONTH] [j/JOB] [a/APPLICATION_STATUS] [s/SKILL]…`<br>e.g.`edit 2 n/James Lee e/jameslee@example.com`, `edit ALL a/REJECTED`
-**Delete** | `delete INDEX`/ `delete ALL`<br> e.g. `delete 3`, `delete ALL`
+**Filter** | `filter [g/GRADE] [i/INSTITUTION]…+ [c/COURSE]…+ [y/GRADUATION_YEAR_MONTH] [j/JOB]..+ [a/STATUS]..+ [s/SKILL]…+` <br> e.g. `filter s/JAVA y/06/2022 i/NUS a/REJECTED`
+**Edit** | `edit ALL a/APPLICATION_STATUS` , <br>`edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [g/GRADE] [i/INSTITUTION] [c/COURSE] [y/GRADUATION_YEAR_MONTH] [j/JOB] [a/APPLICATION_STATUS] [s/SKILL]…`<br>e.g. `edit ALL a/REJECTED` , <br> `edit 2 n/James Lee e/jameslee@example.com`
```
Comment (jackgugz) [inline docs/UserGuide.md:378]: should be edit [ALL] since ALL is optional

Comment (jackgugz) [reply docs/UserGuide.md:378]: best split edit from edit [ALL] command

**Review by jackgugz — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (yunpeng1234) [reply docs/UserGuide.md:378]: ALL is not optional. It is either Index or ALL

Comment (jackgugz) [reply docs/UserGuide.md:378]: Ok I see it now.

**Review by jackgugz — APPROVED** [review]
LGTM

**Pull request merged**